projects/storefrontapp-e2e-cypress/cypress/integration/regression/miscellaneous/scroll-back-to-top.core-e2e-spec.ts

```diff
@@ -0,0 +1,27 @@
+import { visitHomePage } from '../../../helpers/checkout-flow';
```

Comment:
If the e2e doesn't cover more cases than the unit test, we should consider keeping only the unit test and save execution time on the e2e suite.

Reply:
Removed e2e test file